justfile

```diff
@@ -11,12 +11,13 @@ alias c := conformance
 alias f := fix
 alias new-typescript-rule := new-ts-rule
 
-# Make sure you have cargo-binstall installed.
-# You can download the pre-compiled binary from <https://github.com/cargo-bins/cargo-binstall#installation>
-# or install via `cargo install cargo-binstall`
```

Comment:
This should stay somewhere for a hint for missing `cargo binstall` 🤔

Reply:
Sorry for the late reply. I have added the `cargo install cargo-binstall` command to init. Do we still want to keep this?

Reply:
Can you revert? the change is `add pnpm install to init`, not removing this comment.

Reply:
Of course, thanks for your patient answer.